src/Microsoft.OData.Client/ReadingDeltaResourceSetArgs.cs

```diff
@@ -0,0 +1,33 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="ReadingDeltaResourceSetArgs.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+
+namespace Microsoft.OData.Client
+{
+    /// <summary>
+    /// The reading delta resource set arguments
+    /// </summary>
+    public sealed class ReadingDeltaResourceSetArgs
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ReadingDeltaResourceSetArgs" /> class.
+        /// </summary>
+        /// <param name="deltaResourceSet">The OData delta resource set.</param>
+        public ReadingDeltaResourceSetArgs(ODataDeltaResourceSet deltaResourceSet)
+        {
+            Util.CheckArgumentNull(deltaResourceSet, "deltaResourceSet");
+            this.DeltaResourceSet = deltaResourceSet;
+        }
+
+        /// <summary>
+        /// Gets the OData delta resource set.
+        /// </summary>
+        /// <value>
+        /// The OData delta resource set.
+        /// </value>

```

Comment:
I haven't come across this before... Do we need it yet it doesn't add any more information beyond what is in `<summary>`?

Reply:
update this to `ReadingDeltaFeedArgs` and yes it is used.